recipes/squash/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "squash" %}
+{% set version = "0.7.0" %}
+{% set version_major = "0.7" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "4e741b1ab9dff9433fe9dd7393c7db41173b49b0426dcfcd03b13d0b03bc19b8" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.bz2
+  url: https://github.com/quixdb/squash/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.bz2
+  {{ hash_type }}: {{ hash_val }}
+  patches:
+    - patches/0001-Remove-wrongly-placed-break-on-cmake-file.patch
+    - patches/0002-Add-directive-to-have-_SC_PAGE_SIZE-defined.patch
+    - patches/0003-Fix-squash-missing-endian-declarations-in-OSX.patch
+    - patches/0004-Fix-zconf.h-generation.patch
+    - patches/0005-Fix-lzham-compilation-on-OSX.patch
+    - patches/0006-Fix-negative-left-shifting.patch
+    - patches/0007-Fix-negative-bit-left-shifting.patch
+    - patches/0008-Backported-unaligned-operations-patch.patch
+    - patches/0009-Skip-OSX-only-failing-tests-for-now.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - pkg-config
+    - cmake 3.*
+    - ragel >=6.6
+    - glib 2.51.*
+    - bzip2 1.0.*
+    - xz 5.2.*
+    - lzo  # [linux]
+    - snappy
+    - zlib 1.2.*
+
+  run:
+    - glib 2.51.*
+    - bzip2 1.0.*
+    - xz 5.2.*
+    - lzo  # [linux]
```

Comment:
It wasn't at first, but then I noticed the only recipe that uses `lzo` is `pytables` and uses `[linux]` selector. Then noticed that the `osx-64` package is actually empty:

```
$ curl -L https://anaconda.org/anaconda/lzo/2.06/download/osx-64/lzo-2.06-0.tar.bz2 | tar jtvf -
-rw-r--r--  0 ilan   admin       0 Sep 11  2012 .index/lzo-2.06-0
-rw-r--r--  0 ilan   admin      18 Sep 11  2012 info/files
-rw-r--r--  0 ilan   admin       0 Sep 11  2012 info/has_prefix
-rw-r--r--  0 ilan   admin     142 Sep 11  2012 info/index.json
-rw-r--r--  0 ilan   admin     159 Sep 11  2012 info/meta
```

As far I can tell from the build logs, `pytables` builds without `lzo` support and `squash` uses the vendored `lzo` code. 

I have no idea why the `lzo` package is empty for `osx-64`. But stating that we only use it for `linux` seems a good thing to me.